src/rust/Cargo.toml

```diff
@@ -16,15 +16,15 @@ default = ["http"]
 http = ["http-range-client", "bytes", "reqwest"]
 
 [dependencies]
-flatbuffers = "23.5.26"
+flatbuffers = "24.3.25"
```

Comment:
I don't think we should bump flatbuffers in this update, that should be done with new code gen and I prefer to do that for all languages with a single PR.

Reply:
Good point. 

I didn't realize until now that flatbuffers versioning were simply dates, and irrespective of any semantic versioning, so I made our constraint specific to avoid any potential over-eager patch updates that might not be semver compatible.